deepchem/data/data_loader.py

```diff
@@ -419,3 +495,49 @@ def load_img(image_files):
       else:
         raise ValueError("Unsupported image filetype for %s" % image_file)
     return np.array(images)
+
+
+class MolecularComplexLoader(DataLoader):
```

Comment:
Is this a feature you haven't finished implementing yet?

Reply:
To make it cleaner I've removed the stub from the PR and will add a full implementation in a subsequent PR.